newrelic/api/opentelemetry.py

```diff
@@ -208,7 +254,13 @@ def update_name(self, name):
             self.nr_trace.name = self._name
 
     def is_recording(self):
-        return self._sampled() and not (getattr(self.nr_trace, None), "end_time", None)
+        # TODO: Refine this logic along with `_sampled()`
+        if getattr(self.nr_trace, "end_time", None):
```

Comment:
Is there a case where a span has been created but recording hasn't started yet that we need to handle here?